src/wizards/tokenSwapDealWizard/stages/submitStage/submitStage.html

```diff
@@ -137,6 +137,57 @@ <h4 class="submitCardHeader">Terms</h4>
                       <pre class="subtext">${clause.text}</pre>
                     </div>
                   </template>
+                </template>
+
+                <div class="submitContentLabel">DAOplomat Reward</div>
+
+                <div class="submitDaoTokenCard">
+
+                  <pcard class="rewardsCard">
+                    <table class="rewardsTable">
```

Comment:
optional/for the future: I'd prefer css grid for table related styling.
Because here, it is not used as an actual table

Reply:
actually, the HTML looks cleaner a bit with grid. will keep the CSS grid 😄